app/src/main/java/com/example/event_finder_capstone/MainActivity.java

```diff
@@ -1,14 +1,118 @@
 package com.example.event_finder_capstone;
 
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.widget.Toast;
+
+import androidx.annotation.NonNull;
 import androidx.appcompat.app.AppCompatActivity;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
 
-import android.os.Bundle;
+import com.example.event_finder_capstone.models.Event;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.parse.ParseUser;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
 
 public class MainActivity extends AppCompatActivity {
+    RecyclerView rvEvents;
+    private List<Event> eventsList = new ArrayList<>();
+    private EventsAdapter eventsAdapter;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        rvEvents = findViewById(R.id.rvEvents);
+        eventsList = new ArrayList<>();
+        eventsAdapter = new EventsAdapter(this, eventsList);
+        rvEvents.setAdapter(eventsAdapter);
+        LinearLayoutManager linearLayoutManager = new LinearLayoutManager(this);
+        rvEvents.setLayoutManager(linearLayoutManager);
+
+        getAPIEvents();
```

Comment:
🛠 Opportunity:  Thanks for splitting `getAPIEvents()` into a method.  There might be an opportunity for the new code above to be pulled into a method as well.